class/openshift4-monitoring.yml

```diff
@@ -10,6 +10,27 @@ parameters:
           api-usage: https://raw.githubusercontent.com/openshift/cluster-kube-apiserver-operator/release-4.8/bindata/v4.1.0/alerts/api-usage.yaml
           cpu-utilization: https://raw.githubusercontent.com/openshift/cluster-kube-apiserver-operator/release-4.8/bindata/v4.1.0/alerts/cpu-utilization.yaml
           slos: https://raw.githubusercontent.com/openshift/cluster-kube-apiserver-operator/release-4.8/bindata/v4.1.0/alerts/kube-apiserver-slos.yaml
+        release-4.9:
+          api-usage: https://raw.githubusercontent.com/openshift/cluster-kube-apiserver-operator/release-4.9/bindata/assets/alerts/api-usage.yaml
+          cpu-utilization: https://raw.githubusercontent.com/openshift/cluster-kube-apiserver-operator/release-4.9/bindata/assets/alerts/cpu-utilization.yaml
+          slos: https://raw.githubusercontent.com/openshift/cluster-kube-apiserver-operator/release-4.9/bindata/assets/alerts/kube-apiserver-slos-basic.yaml
+      elasticsearch-operator:
+        release-4.7:
+          prometheus: https://raw.githubusercontent.com/openshift/elasticsearch-operator/${openshift4_monitoring:manifests_version}/hack/prometheus-rules.yaml
+        release-4.8:
+          prometheus: https://raw.githubusercontent.com/openshift/elasticsearch-operator/${openshift4_monitoring:manifests_version}/hack/prometheus-rules.yaml
+        release-4.9:
+          # Release 4.9 was forgotten but 5.0 and 4.8 are identical
```

Comment:
Side-note: The ElasticSearch operator releases don't necessarily match OCP releases (cf. component openshift4-logging).

Reply:
We most likely should move the rules there https://github.com/appuio/component-openshift4-logging.

Reply:
yeah, I thought there was an issue for that somewhere already, but I can't find it right now. We should also have #36 before doing that so we don't have to copy-paste the whole rule patching into the logging component.